test/fail_compilation/checkimports3.d

```diff
@@ -0,0 +1,15 @@
+/*
+REQUIRED_ARGS: -transition=checkimports -de
+TEST_OUTPUT:
+---
+fail_compilation/checkimports3.d(14): Deprecation: local import search method found overloadset checkimports3.foo (3 overloads) instead of overloadset checkimports3.foo (2 overloads)
```

Comment:
Because it tells you that there is an actual difference, you have to manually inspect those sematic changes anyhow reported with -transition=checkimports.